src/sentry/static/sentry/app/routes.jsx

```diff
@@ -743,13 +742,22 @@ function routes() {
       <Route path="/:orgId/" component={errorHandler(OrganizationDetails)}>
         <Route component={errorHandler(OrganizationRoot)}>
           <IndexRoute component={errorHandler(OrganizationDashboard)} />
-
           <Route
             path="/organizations/:orgId/discover/"
-            componentPromise={() =>
-              import(/*webpackChunkName:"OrganizationDiscover"*/ './views/organizationDiscover/index')}
-            component={errorHandler(LazyLoad)}
+            /* We do not lazy load discover since it causes the components
```

Comment:
Hmm if I had to choose between the two, I'd rather have Discover lazy loaded. 

Would it work if we keep it lazy loaded, and remove `component` from the saved query route?

Reply:
Ok actually this seems to work with nesting the routes